content/id/docs/tutorials/kubernetes-basics/update/update-intro.html

```diff
@@ -0,0 +1,135 @@
+                        </div>
+
+                        <div class="item carousel-item">
+                          <img src="/docs/tutorials/kubernetes-basics/public/images/module_06_rollingupdates2.svg">
+                        </div>
+
+                        <div class="item carousel-item">
+                          <img src="/docs/tutorials/kubernetes-basics/public/images/module_06_rollingupdates3.svg">
+                        </div>
+
+                        <div class="item carousel-item">
+                          <img src="/docs/tutorials/kubernetes-basics/public/images/module_06_rollingupdates4.svg">
+                        </div>
+                      </div>
+
+                      <a class="left carousel-control" href="#myCarousel" role="button" data-slide="prev">
+                        <span class="sr-only ">Sebelumnya</span>
+                      </a>
+                      <a class="right carousel-control" href="#myCarousel" role="button" data-slide="next">
+                        <span class="sr-only">Selanjutnya</span>
+                      </a>
+
+                    </div>
+            </div>
+        </div>
+        <br>
+
+        <div class="row">
+            <div class="col-md-8">
+
+                <p>Seperti penskalaan aplikasi, jika suatu Deployment terekspos secara publik, maka Service akan me-<i>load-balance</i> trafik hanya ke Pod yang tersedia saat pembaruan berlangsung. Pod yang tersedia adalah instans yang dapat diakses oleh pengguna aplikasi.</p>
+
+                <p>Pembaruan bertahap memungkinan dilakukannya aktivitas-aktivitas berikut:</p>
+                <ul>
+                    <li>Mempromosikan suatu aplikasi dari satu <i>environment</i> yang ke <i>environment</i> yang lain (via pembaruan kontainer <i>image</i>)</li>
+                    <li>Kembali (<i>rollback</i>) ke versi-versi sebelumnya</li>
+                    <li>Continuous Integration dan Continuous Delivery (CI/CD) dari aplikasi tanpa waktu henti</li>
+                </ul>
+
+            </div>
+            <div class="col-md-4">
+                <div class="content__box content__box_fill">
+                    <p><i>Jika suatu Deployment terekspos secara publik, maka Service akan me-<i>load-balance</i> trafik hanya ke Pod yang tersedia saat pembaruan berlangsung.</i></p>
+                </div>
+            </div>
+        </div>
+
+        <br>
+
+        <div class="row">
+            <div class="col-md-8">
+                <p>Pada tutorial interaktif berikut ini, kita akan mencoba untuk memperbarui aplikasi kita ke versi terbaru dan kemudian melakukan <i>rollback</i>.</p>
+            </div>
+        </div>
+        <br>
+
+        <div class="row">
+            <div class="col-md-12">
+                <a class="btn btn-lg btn-success" href="/docs/tutorials/kubernetes-basics/update/update-interactive/" role="button">Mulai Tutorial Interaktif <span class="btn__next">›</span></a>
```

Comment:
change to ID doc if available.
```suggestion
                <a class="btn btn-lg btn-success" href="/id/docs/tutorials/kubernetes-basics/update/update-interactive/" role="button">Mulai Tutorial Interaktif <span class="btn__next">›</span></a>
```